Clarify description of repository (#63)
publiccodenet/publiccode.net: README.md

# Homepage at [publiccode.net](https://publiccode.net)

> This website is the homepage for the Foundation for Public Code. It is where we introduce and explain ourselves to the outside world next to being a portal to the content that the Foundation for Public Code has to offer.

Reviewer: ```suggestion
This website is the homepage for the Foundation for Public Code. It is where we introduce and explain ourselves to the outside world, and invite readers into all the content that the Foundation for Public Code has to offer.
```
